website/docs/d/availability_zones.html.markdown

```diff
@@ -38,21 +40,44 @@ resource "aws_subnet" "secondary" {
 }
 ```
 
+### By Filter
```

Comment:
Would it be worthwhile to add a snippet along the lines of "so you've enabled Local Zones, here's how to exclude them from your AZ list"? It might help avoid unexpected behaviour.